cmd/qliksense/apply.go

```diff
@@ -1,39 +1,39 @@
 package main
 
 import (
-	"os"
+	"io"
 
-	"github.com/pkg/errors"
 	"github.com/qlik-oss/sense-installer/pkg/qliksense"
 	"github.com/spf13/cobra"
 )
 
 func applyCmd(q *qliksense.Qliksense) *cobra.Command {
-
+	opts := &qliksense.InstallCommandOptions{}
 	filePath := ""
+	keepPatchFiles := false
+	overwriteExistingContext := false
 	c := &cobra.Command{
 		Use:     "apply",
 		Short:   "install qliksense based on provided cr file",
 		Long:    `install qliksense based on provided cr file`,
-		Example: `qliksense apply -f file_name or cat cr_file | qliksense apply -f -`,
```

Comment:
since you added pipe we can keep this example

Reply:
addressed